.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Checklist:
+
+- [ ] Correct branch: master for new features / 3.2.x, 3.1.x for fixes
```

Comment:
what about `;` instead of `/` ?

Just because slash used below is used to group 3.2 and master, but here it separates 3.2 and master